docs/UsersGuide/dev/baremetal-multicore.md

```diff
@@ -164,7 +164,7 @@ the next. Parallelism has been virtualized and the processor is shareable withou
 nor requiring processor instruction set support to switch threading contexts.
 
 Inside F´ a parallel implementation of the active component task was implemented such that it returns rather than blocks
-on receiving messages. When `BAREMERAL_SCHEDULER` is enabled in the F´ configuration, this alternate implementation is
+on receiving messages. When `BAREMETAL_SCHEDULER` is enabled in the F´ configuration, this alternate implementation is
```

Comment:
This was caught when I had temporarily omitted the current `patterns.txt` as part of the refresh...